DOCSP-38088 added the v1.18 release notes (#346)
mongodb/docs-atlas-cli: source/atlas-cli-changelog.txt

   :depth: 1
   :class: singlecol

.. _atlas-cli-1.18.0:

{+atlas-cli+} 1.18.0
--------------------

Released 27 March 2024

- Fixes a bug in the :ref:`atlas-events-projects-list`,
  :ref:`atlas-projects-list` and :ref:`atlas-events-organizations-list`
  commands where an invalid ``minDate`` and ``maxDate`` silently failed
  and didn't apply to the command.
- Fixes an issue where the {+atlas-cli+} didn't sign Windows binaries.
- Fixes an issue where the {+atlas-cli+} didn't apply the correct
  default MongoDB version for {+cluster+} creation.
- Fixes a bug where the :ref:`atlas-customDbRoles-create` command fails

>   if you provide more than one privilege with same action.

Reviewer: ```suggestion
  if you provide more than one privilege with the same action.
```

   :depth: 1
   :class: singlecol

.. _atlas-cli-1.18.0:

{+atlas-cli+} 1.18.0
--------------------

Released 27 March 2024

- Fixes a bug in the :ref:`atlas-events-projects-list`,
  :ref:`atlas-projects-list` and :ref:`atlas-events-organizations-list`
  commands where an invalid ``minDate`` and ``maxDate`` silently failed
  and didn't apply to the command.
- Fixes an issue where the {+atlas-cli+} didn't sign Windows binaries.
- Fixes an issue where the {+atlas-cli+} didn't apply the correct
  default MongoDB version for {+cluster+} creation.

> - Adds a note on the :ref:`atlas-clusters-update` command to warn that
>   {+atlas-cli+} doesn't allow upgrades from replica set to multi-sharded
>   nodes.
> - Adds a notification when a session expires and about best practices to
>   use the {+atlas-cli+} programatically.

Reviewer 2: these are copy changes, no need for release notes
```suggestion
```

> - Fixes a bug where creating a ``customDbRoles`` fails if you provide

Reviewer: ```suggestion
- Fixes a bug where creating a custom database role fails if you provide
```
I understand that you're referring to the command syntax here, but the 'a' in front of the pluralized wording of the command feels a bit awkward. Since this corresponds to an object called a custom database role, we could use that, allowing us the option of the singular form. Alternatively, if you want to keep the command syntax in place, perhaps  "Fixes a bug where ``atlas customDbRoles create`` fails . . ." ?
